conda-recipe/meta.yaml

```
@@ -1,6 +1,6 @@
 {% set package_name = "btms_ui" %}
 {% set import_name = "btms_ui" %}
-{% set version = load_file_regex(load_file=os.path.join(import_name, "_version.py"), regex_pattern=".*version = '(\S+)'").group(1) %}
+{% set version = load_file_regex(load_file=os.path.join(import_name, "version.py"), regex_pattern=".*version = '(\S+)'").group(1) %}
```

```suggestion
{% set version = load_file_regex(load_file=os.path.join(import_name, "_version.py"), regex_pattern=".*version = '(\S+)'").group(1) %}
```

Same as above, somewhat unintuitively